cmd/internal/cli/singularity.go

```diff
@@ -580,3 +580,18 @@ func singularityExec(image string, args []string) (string, error) {
 
 	return stdout.String(), nil
 }
+
+// CheckRoot ensures that a command is executed with root privileges.
+func CheckRoot(cmd *cobra.Command, args []string) {
+	if os.Geteuid() != 0 {
+		sylog.Fatalf("%q command requires root privileges", cmd.CommandPath())
+	}
+}
+
+// CheckRootAndInstall ensures that a command is executed with root
+// privileges or that Singularity is installed unprivileged.
+func CheckRootAndInstall(cmd *cobra.Command, args []string) {
```

Comment:
`CheckRootAndInstall` is a bit unclear to me, until I look at this comment on the `func` def. 

What about `CheckRootOrUnpriv` - as that suggests more that we are checking if we are root _or_ singularity is installed unpriv?

Reply:
Yes that's better, thanks @dctrud :)